apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -14,7 +14,7 @@ type Properties<T extends FieldValues> = {
 	autoComplete?: string;
 	control: Control<T, null>;
 	errors: FieldErrors<T>;
-	label: string;
+	label?: string;
```

Comment:
Label must be required

Reply:
I don't need label in my search field. And if I use an empty string the empty space is appeared and styles brokes) I didn't find a way to resolve it except for this one.